public/locales/en/translation.json

```diff
@@ -436,13 +435,18 @@
             },
             "welcome": {
                 "text": {
-                    "bugs": "Please note that HyperPlay is a public alpha. Many features are still\n          in development, and there will be bugs.",
-                    "communityCTA_1": "We'd love your feedback and to have you join us in our",
-                    "communityCTA_2": "community. Together, let's shape the future of gaming!",
-                    "construction": "HyperPlay is under construction.",
-                    "description": "HyperPlay is a game launcher and game store aggregator from the\n          future. With HyperPlay, you can carry your wallet, tokens, and assets\n          into every game. HyperPlay supports the entire library of the Epic\n          Store, GOG, and our own HyperPlay store. By using HyperPlay, you agree\n          to our"
+                    "2": "Currently in Beta, we highly value your feedback and \n            encourage you to report any issues or suggestions in our",
+                    "bulletPoint_1": "Access a vast collection of 100+ on-chain enabled games",
+                    "bulletPoint_2": "Earn tokens, NFTs, and off-chain rewards by completing Quests",
+                    "bulletPoint_3": "Play Epic and GOG games directly through HyperPlay",
+                    "bulletPoint_4": "Install Steam (Windows version) on macOS with one easy click",
+                    "bulletPoint_5": "Run Windows games on macOS, Linux, and SteamDeck using our\n              advanced",
+                    "bulletTitle": "With HyperPlay, you can:",
+                    "communityCTA_4": "Let's shape the future of gaming together!",
```

Comment:
The i18n workflow error is related with this line here. Running locally it gets removed since it is not used anywhere.
So try again to run `yarn i18n` and commit the change.